src/main/java/com/uid2/admin/Main.java

```diff
@@ -195,6 +196,23 @@ public void run() {
             operatorKeyProvider.loadContent(operatorKeyProvider.getMetadata());
             OperatorKeyStoreWriter operatorKeyStoreWriter = new OperatorKeyStoreWriter(operatorKeyProvider, fileManager, jsonWriter, versionGenerator);
 
+            CloudPath s3KeyMetadataPath = new CloudPath(config.getString(Const.Config.S3keysMetadataPathProp));
+            GlobalScope s3KeyGlobalScope = new GlobalScope(s3KeyMetadataPath);
+            RotatingS3KeyProvider s3KeyProvider = new RotatingS3KeyProvider(cloudStorage, s3KeyGlobalScope);
+            S3KeyStoreWriter s3KeyStoreWriter = new S3KeyStoreWriter(s3KeyProvider, fileManager, jsonWriter, versionGenerator, clock, s3KeyGlobalScope);
+            S3KeyManager s3KeyManager = new S3KeyManager(s3KeyProvider, s3KeyStoreWriter);
+            s3KeyManager.generateKeysForOperators(operatorKeyProvider.getAll(), config.getLong("s3_key_activates_in_seconds"), config.getInteger("s3_key_count_per_site"));
+            try {
+                s3KeyProvider.loadContent(s3KeyProvider.getMetadata());
```

Comment:
I want to change  s3KeyProvider.loadContent(s3KeyProvider.getMetadata()); to  s3KeyProvider.loadContent(), need to edit share though. seems like too late